src/site/content/en/blog/variable-fonts/index.md

```diff
@@ -0,0 +1,359 @@
+---
+title: Introduction to variable fonts on the web
+subhead: We will look at what variable fonts are, how we can use them in our work.
+authors:
+  - mustafakurtuldu
+  - thomassteiner
+date: 2018-02-19
+updated: 2020-05-12
+hero: hero.jpg
+tags:
+  - blog
+  - fonts
+---
+In this article, we will look at what variable fonts are, how we can use them in
+our work, and the potential possibilities they entail. But to understand what
+they offer, first we must explore how typography and font loading currently
+work on the web.
+
+## Introduction
+
+The terms font and typeface are used interchangeably in the developer industry.
+However there is a difference: A typeface includes an entire family of designs,
+such as the [Roboto](https://fonts.google.com/specimen/Roboto) typeface.
+Meanwhile a font is one of the digital files of that family, like "Roboto Bold"
+or "Roboto Italic." In other words, a typeface is what you _see_, and the font is
+what you _use_.
+
+<figure class="w-figure">
+  <img src="robot-specimen.png" alt="A specimen of Roboto">
+  <figcaption class="w-figcaption">
+    Roboto was designed and developed by Christian Robertson.
+    Depicted is a specimen of the Roboto typeface.
+  </figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="roboto-family.png" alt="Different members of the Roboto family">
+  <figcaption class="w-figcaption">
+    Different fonts within the Roboto family.
+  </figcaption>
+</figure>
+
+### Challenges for the Designer and Developer
+
+When a graphic designer prepares their work, they typically export the final
+artwork in a way that either embeds all of the fonts used, or they provide
+an export archive that contains all of the assets used separately, a bit like a
+basic website. The cost to the designer depends on the suitability of the
+typeface for the rendering context  (is it legible at small sizes, for example)
+and the license to use the fonts.
+
+On the web, we have to consider both aspects, plus the associated bandwidth
+costs. For every member of a typeface family used in our designs, we have had to
+require another font file to be downloaded by our users before they can see that
+text. Only including the Regular and Bold styles, plus their italic
+counterparts, can amount to 500k or more of data. This has been a sticking point
```

Comment:
```suggestion
counterparts, can amount to 500&nbsp;KB or more of data. This has been a sticking point
```
https://developers.google.com/style/units-of-measure#spaces-in-units-of-measurement